src/nuklear_button.c

```diff
@@ -198,7 +198,12 @@ nk_draw_button_symbol(struct nk_command_buffer *out,
     else if (state & NK_WIDGET_STATE_ACTIVED)
         sym = style->text_active;
     else sym = style->text_normal;
-    nk_draw_symbol(out, type, *content, bg, sym, 1, font);
+
+	/* don't let symbol escape button bounds */
+	struct nk_rect symbol_content = *content;
+	float f = NK_MIN(symbol_content.h, symbol_content.w);
```

Comment:
Hm, thinking about this a bit - do we need to force symbols to be rectangular?

Reply:
Dunno.  I do know the existing behavior is not good.

Reply:
@hejsil thoughts?

Reply:
I'm guessing symbols somehow ends up escaping the bounds of the button (a picture would be nice so I had an idea as to how this failure looks). This rectangle solution seems wrong to me tough. Shouldn't we just shrink the content by `style->button.border`? Ofc, this wont solve anything if the symbol clips at the corners and the corners are rounded.

Reply:
Shrinking sounds also reasonable (round corners are theoretically solvable by changing the symbol itself, so probably not a big issue).